docs/cloud/pouta/vm-flavors-and-billing.md

```diff
@@ -315,16 +313,31 @@ performance is significantly better.
 
 **io.\*:**
 
+NOTE! These servers have non-redudnant disks, and you may expect data loss in case of disk failure.
```

Comment:
redundant.
Also maybe good to note that io.* is 70GB - 700GB, so that its clear that its just the first gen since we didn't start with io.1. Maybe